tool/ircproxy-privmsg-starttls.py

```diff
@@ -1,522 +1,522 @@
-#!/usr/bin/env python
```

Comment:
I'm seeing a complete file changeover, don't know why?  Did you change tabs to spaces?  This is difficult to process...